.vitepress/config/en.ts

```diff
@@ -23,7 +23,7 @@ export const enConfig = defineLocaleConfig("root", {
       {
         text: "Resources",
         items: [
-          { text: "Blog", link: "/blog/2025-03-15-oxlint-beta" },
+          { text: "Blog", link: "/blog/2025-06-01-oxlint-stable" },
```

Comment:
Replaced? Need to add another entry.

Reply:
ah sorry i meant the v1 release post rather than a different post

Reply:
> this should point to the v1 release post rather than a different post

correct